pom.xml

```diff
@@ -106,9 +105,9 @@ under the License.
 
     <eclipse-sisu.version>0.9.0.M2</eclipse-sisu.version>
     <guice.version>6.0.0</guice.version>
-    <lucene.version>9.8.0</lucene.version>
+    <lucene.version>9.9.0</lucene.version>
     <maven.version>3.9.5</maven.version>
```

Comment:
@mbien could you up this to 3.9.6 please as well? so have it all at once?

Reply:
roger, will update and force push in a sec